index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PortFolio Project</title>
+
+  <!-- Linking the page to the style file A.css -->
+  <link rel="stylesheet" href="styles/style.css">
+  
+</head>
+<body>
+  <header class="toolbar">
+    <section class="logo">
+      MZK
+    </section>
+    <section class="menu">
+      <img src="images/Union.png" alt="">
+    </section>
+  </header>
+
+  <!-- Headline section -->
+  <section class="headline">
+    <h1>
+        Hey there. <br>I’m Mzirkof .
+      <p>
+          I’m a software developer
+      </p>
+    </h1>
+    <p class="description">
+      I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hesitate to contact me.
+    </p>
+    <section class="social">
+      <img src="images/twitter.png" alt="">
+      <img src="images/Linkedin icon.png" alt="">
+      <img src="images/medium.png" alt="">
+      <img src="images/github.png" alt="">
+      <img src="images/angellist.png" alt="">
+    </section>
```

Comment:
- Good job done so far, but to help improve the semantics of your site, please use the `ul` semantic tag to wrap this section and ensure every image is wrapped in a corresponding `li` tag. Please fix this 🙏